backend/database_handler/transactions_processor.py

```diff
@@ -182,3 +183,19 @@ def get_transaction_count(self, address: str) -> int:
             .count()
         )
         return count
+
+    def get_transactions_for_address(self, address: str) -> list[dict]:
+        transactions = (
+            self.session.query(Transactions)
+            .filter(
+                or_(
+                    Transactions.from_address == address,
+                    Transactions.to_address == address,
+                )
+            )
+            .all()
+        )
```

Comment:
I'm still thinking if we should only check for `to_address`, since that would give us all the ordered `nonce`s and it's probably what we want. WDYT?

Reply:
in the explorer you get both at the same time sorted by date. Maybe you can include an optional parameter to filter by only_from or only_to

Reply:
@cristiam86 I've added the filter :+1: